### PR TITLE
Integrate new version of klee

### PR DESCRIPTION
**Pull request by sava-cska**

### CODE REVIEW

File: server/test/framework/Syntax_Tests.cpp [hunk 1]
```diff
@@ -3601,4 +3603,69 @@ namespace {
...
+                  },
+                  [](const tests::Tests::MethodTestCase &testCase) {
+                      return stoi(testCase.returnValue.view->getEntryValue(nullptr)) == -3;
+                  },
+                  [](const tests::Tests::MethodTestCase &testCase) {
+                      return stoi(testCase.returnValue.view->getEntryValue(nullptr)) == 3;
+                  } }));
+    }
+
+    TEST_F(Syntax_Test, Run_Tests_For_Hard_Linked_List) {
```
Comment (ladisgin) [inline server/test/framework/Syntax_Tests.cpp:3635]: run tests must be in another test case

File: server/src/Tests.cpp [hunk 2]
```diff
@@ -681,18 +701,52 @@ void KTestObjectParser::assignTypeStubVar(Tests::MethodTestCase &testCase,
...
+                                  testCase.lazyReferences, methodDescription);
             testCase.stubParamValues.emplace_back(obj.name, 0, stubView);
             testCase.stubParamTypes.emplace_back(stubType, obj.name, std::nullopt);
         }
     }
 }
 
+void KTestObjectParser::assignAllLazyPointers(
+    Tests::MethodTestCase &testCase,
+    const std::vector<std::optional<Tests::TypeAndVarName>> &objects) const {
```
Comment (ladisgin) [inline server/src/Tests.cpp:719]: rename objects

**Review by ladisgin — CHANGES_REQUESTED** [review]

File: server/test/framework/Syntax_Tests.cpp (hunk 1 above)
Comment (sava-cska) [reply server/test/framework/Syntax_Tests.cpp]: Fix

File: server/src/Tests.cpp (hunk 2 above)
Comment (sava-cska) [reply server/src/Tests.cpp]: Fix

**Review by ladisgin — APPROVED** [review]

**Pull request merged**